### PR TITLE
parse table generator is now packing terminal parameters into tuple

### PR DESCRIPTION
**Pull request by wieczyk**
Fix for #179.

### CODE REVIEW

File: lalrpop-test/src/expr.lalrpop [hunk 1]
```diff
@@ -11,6 +11,7 @@ extern {
         "*" => Tok::Times,
         "/" => Tok::Div,
         Num => Tok::Num(<i32>),
+        Fraction => Tok::Fraction(<i32>, <i32>),
```
Comment (nikomatsakis) [inline lalrpop-test/src/expr.lalrpop]: Is this a test that reproduces the problem? (or used to, before your fix)

Comment (wieczyk) [inline lalrpop-test/src/expr.lalrpop]: Yes.

The presence of this declaration is enough to provoke error.

Comment (nikomatsakis) [inline lalrpop-test/src/expr.lalrpop]: Can you add a comment like `// Regression test for #179` or something so that we remember that?

Comment (wieczyk) [inline lalrpop-test/src/expr.lalrpop]: Sure.

**Comment by nikomatsakis** [comment]
Thanks =)

**Pull request merged**